Server/mods/deathmatch/logic/CUnoccupiedVehicleSync.cpp

```diff
@@ -417,6 +417,14 @@ void CUnoccupiedVehicleSync::Packet_UnoccupiedVehicleSync(CUnoccupiedVehicleSync
                         bool bDerailed = pVehicle->IsDerailed();
                         bool bInWater = pVehicle->IsInWater();
 
+                        if (bInWater && bInWater != pVehicle->GetLastSyncedIsInWater())
+                        {
+                            // Call the event once in water
+                            CLuaArguments Arguments;
+                            pVehicle->CallEvent("onVehicleDrown", Arguments);
```

Comment:
This is a convention in the code more or less, afaik.
But I'd use `{}` there, or `CLuaArguments()` (but `{}` should work)
Not even sure why CallEvent doesn't default construct `CLuaArguments`